stdlib/public/core/Sendable.swift

```diff
@@ -10,13 +10,103 @@
 //
 //===----------------------------------------------------------------------===//
 
-/// The Sendable protocol indicates that value of the given type can
-/// be safely used in concurrent code.
+/// A type whose values can safely be passed across concurrency domains by copying.
+///
+/// You can safely pass values of a sendable type
+/// from one concurrency domains to another ---
+/// for example, you can pass a sendable value as the argument
+/// when calling an actor's methods.
+/// All of the following can be marked as sendable:
+///
+/// - Value types
+/// - References to immutable reference types
```

Comment:
This reads a bit like we're marking the references themselves rather than the types. How about:

```suggestion
/// - Reference types with no mutable storage
```